### PR TITLE
doc: update requirements

### PR DESCRIPTION
**Pull request by vivianjeng**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally and any changes were pushed
- [x] Lint (`yarn lint --check`) has passed locally and any fixes were made for failures

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

- update requirements in readme
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,11 @@ For more information, refer to the [documentation](https://docs.unirep.io/)
     | [`utils`](./packages/utils) | <a href="https://www.npmjs.com/package/@unirep/utils"><img alt="NPM version" src="https://img.shields.io/npm/v/@unirep/utils?color=%230004&style=flat-square" /></a> | Utils which are used in unirep protocol. |
     | [`subgraph`](./packages/subgraph/) | - | Subgraph of unirep smart contract. |
 ​
+## 🔋 Requirements
+
+- Intall [rust](https://www.rust-lang.org/tools/install) and [circom 2](https://docs.circom.io/getting-started/installation/)
```
Comment (vimwitch) [inline README.md:47]: This dependency is only for people changing the circuits. I think application developers should be able to use unirep without circom.

File: README.md [hunk 2]
```diff
@@ -42,6 +42,11 @@ For more information, refer to the [documentation](https://docs.unirep.io/)
...
+- Node.js >=16.14
```
Comment (vimwitch) [inline README.md:48]: Can we mark this just `Node.js >= 16`?

**Review by vimwitch — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vivianjeng) [reply README.md:47]: This is not for application developers. This is for developers who want to contribute to the repo and run the repo locally.
The one for application developers is the readme file in each package and the documentation.

File: README.md (hunk 2 above)
Comment (vivianjeng) [reply README.md:48]: This is the requirements for docusaurus
Also I think it is for contributors of this repo so if they want to run `yarn build` locally they will need ` Node.js >=16.14`

Comment (vimwitch) [reply README.md:48]: Huh, didn't know that.

File: README.md (hunk 1 above)
Comment (vimwitch) [reply README.md:47]: > This is not for application developers. This is for developers who want to contribute to the repo and run the repo locally.

I sorta disagree here. A lot of the docs are based around giving application developers an API reference to use when writing unirep applications. The readme's aren't comprehensive enough to help write an application (without having to look at the code).

Why not add a not here saying that rust is only needed for compiling circuits?

Comment (vimwitch) [reply README.md:47]: Oh wait this is the readme for the root repo 🤦, these requirements are fine

**Review by vimwitch — APPROVED** [review]

**Pull request merged**